bot/bot.py

```diff
@@ -119,9 +119,29 @@ def handle_comment(self, comment, strict_match=True):
             if comic is None:
                 continue
 
+            self.database.add_id(comment_id, comic["num"])
             response = self.format_response(comic)
             responses.append(response)
+
+        seen = set()
+        for comic_title in comic_titles:
+            if len(responses) > RESPONSE_COUNT_LIMIT:
+                logger.warning(
+                    f"Exceeded the reponse count limit of {RESPONSE_COUNT_LIMIT} responses")
+                break
+
+            comic = self.get_comic_by_title(comic_title)
+
+            if comic is None:
+                continue
+            elif str(comic["num"]) in comic_ids or str(comic["num"]) in seen:     # check if comic is a duplicate
+                continue
+
+            seen.add(str(comic["num"]))
+
             self.database.add_id(comment_id, comic["num"])
+            response = self.format_response(comic)
+            responses.append(response)
```

Comment:
These lines fix 8a8caa1860828c5c0140a30b8b9efedd4969acd0. The problem was that we were calling **format_response** before adding the new comment in the database, which leads to a **division by zero** error if there are no other comments in the database.

This prevents the error and also produces the correct percentage. Whereas the current implementation in master, doesn't take into account the latest comment when getting total_count from the database.

No need to change anything. I just wanted to point that out.